src/UnitTests/test_quote_prov.cpp

```diff
@@ -33,26 +39,26 @@ static sgx_ql_set_logging_function_t sgx_ql_set_logging_function;
 
 static void Log(sgx_ql_log_level_t level, const char* message)
 {
-    char const * levelText = "ERROR";
-    switch (level) {
+    char const* levelText = "ERROR";
+    switch (level)
+    {
         case SGX_QL_LOG_WARNING:
             levelText = "WARNING";
             break;
         case SGX_QL_LOG_INFO:
             levelText = "INFO";
             break;
         default:
-	    levelText = "ERROR";
+            levelText = "ERROR";
     }

```

Comment:
A break in this default case should be good enough because the default value of levelText was initialized to ERROR.